README.md

```diff
@@ -269,7 +269,7 @@ XCUITest driver supports the following element attributes:
 Name | Description | Example
 --- | --- | ---
 name | Could contain either element's [identifier](https://developer.apple.com/documentation/xctest/xcuielementattributes/1500981-identifier?language=objc) or its [label](https://developer.apple.com/documentation/xctest/xcuielementattributes/1500692-label?language=objc), depending on which one is available first. Could also be `null`. It is recommended to prefer the usage of [accessibilityIdentifier](https://developer.apple.com/documentation/uikit/uiaccessibilityidentification/1623132-accessibilityidentifier) over [accessibilityLabel](https://developer.apple.com/documentation/objectivec/nsobject/1615181-accessibilitylabel) for automation purposes, since the `identifier` property is supposed to stay constant under different locales and does not affect accessibility services such as VoiceOver. | 'hello'
-label | Element's [label](https://developer.apple.com/documentation/xctest/xcuielementattributes/1500692-label?language=objc) value. Could be `null` | 'hello'
+label | Element's [label](https://developer.apple.com/documentation/xctest/xcuielementattributes/1500692-label?language=objc) value. Could be `null`. The result started including `\n` as same as the `label` attribue in the XCUIElement since XCUITest driver 4.7.3 (WebDriverAgent 4.8.0) if the label included it. | 'hello', 'hello\nworld'
```

Comment:
Since XCUIDriver 4.7.3 the behaviour of this value was better aligned with XCTest, so it could include line breaks (`\n`). Before this version line breaks were replaced by spaces.